mobile/components/DataEntry.tsx

```diff
@@ -121,28 +163,56 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: '#fef4e1',
-    justifyContent: 'center'
+    justifyContent: 'center',
+    maxWidth: '89%'
   },
-  smallButton: {
+  rowContainer: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    padding: 10
```

Comment:
I know its early but (especially cause we're on mobile) we gotta keep it responsive with rem rather than px measurements. You can convert them [here!](https://nekocalc.com/px-to-rem-converter)